Heejung/Lv2/소수 찾기.js

```diff
@@ -0,0 +1,37 @@
+// 에라토스테네스의 체
+function makePrimeNums(n) {
+  const arr = new Array(n + 1).fill(true);
+  arr[0] = false;
+  arr[1] = false;
+
+  for (let i = 2; i * i <= n; i += 1) {
+    if (arr[i]) {
+      for (let j = i * i; j <= n; j += i) {
+        arr[j] = false;
+      }
+    }
+  }
+  return arr.map((e, i) => (e ? i : false));
+}
+
+function solution(numbers) {
+  let answer = 0;
+  // numbers 배열로 변환, 내림차순 정렬 => 최댓값
+  const max = numbers.split('').sort((a, b) => b - a);
+  // 소수 구하기
+  const primeNums = makePrimeNums(Number(max.join('')))
+    .filter((e) => e)
+    .map((e) => String(e));
+
+  for (let primeNum of primeNums) {
+    const tmp = primeNum.split('');
+    for (let str of max) {

```

Comment:
```suggestion
  for (const primeNum of primeNums) {
    const tmp = primeNum.split('');
    for (const str of max) {
```
for of 문 안에서도 바뀌지 않는 변수는 const로 선언해주시는게 의도하지 않은 오류를 줄일 수 있어요.